static/app/views/issueList/searchBar.spec.tsx

```diff
@@ -0,0 +1,303 @@
+    defaultProps = {
+      organization,
+      query: '',
+      tagValueLoader: () => tagValuePromise,
+      supportedTags,
+      onSearch: jest.fn(),
+      onSidebarToggle: () => {},
+      sort: 'date',
+    };
+
+    recentSearchMock = MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/recent-searches/',
+      method: 'GET',
+      body: [],
+    });
+  });
+
+  afterEach(function () {
+    MockApiClient.clearMockResponses();
+  });
+
+  describe('updateAutoCompleteItems()', function () {
+    it('sets state with complete tag', function () {
+      const loader = jest.fn();
+
+      render(
+        <OrganizationContext.Provider value={organization}>
+          <IssueListSearchBar {...defaultProps} tagValueLoader={loader} />
+        </OrganizationContext.Provider>,
+        {context: routerContext}
+      );
+
+      userEvent.click(screen.getByRole('textbox'));
+      userEvent.keyboard('url:"fu"');
+
+      expect(loader).toHaveBeenCalledWith('url', 'fu');
+
+      expect(screen.getByTestId('smart-search-dropdown')).toBeInTheDocument();
+      expect(screen.queryByRole('listitem')).not.toBeInTheDocument();
+    });
+
+    it('sets state when value has colon', function () {
+      const loader = jest.fn();
+
+      render(
+        <OrganizationContext.Provider value={organization}>
+          <IssueListSearchBar {...defaultProps} tagValueLoader={loader} />
+        </OrganizationContext.Provider>,
+        {context: routerContext}
+      );
+
+      userEvent.click(screen.getByRole('textbox'));
+      userEvent.keyboard('url:');
+
+      expect(loader).toHaveBeenCalledWith('url', '');
+    });
+
+    it('does not request values when tag is `timesSeen`', function () {
+      // This should never get called
+      const loader = jest.fn(x => x);
+
+      render(
+        <OrganizationContext.Provider value={organization}>
+          <IssueListSearchBar {...defaultProps} tagValueLoader={loader} />
+        </OrganizationContext.Provider>,
+        {context: routerContext}
+      );
+
+      userEvent.click(screen.getByRole('textbox'));
+      userEvent.keyboard('timesSeen:');
+
+      expect(loader).not.toHaveBeenCalled();
+    });
+  });
+
+  describe('Recent Searches', function () {
+    it('saves search query as a recent search', function () {
+      const saveRecentSearch = MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/recent-searches/',
+        method: 'POST',
+        body: {},
+      });
+      const loader = jest.fn();
+      const onSearch = jest.fn();
+
+      render(
+        <OrganizationContext.Provider value={organization}>
+          <IssueListSearchBar
+            {...defaultProps}
+            tagValueLoader={loader}
+            onSearch={onSearch}
+          />
+        </OrganizationContext.Provider>,
+        {context: routerContext}
+      );
+
+      userEvent.click(screen.getByRole('textbox'));
+      userEvent.keyboard('url:"fu"');
```

Comment:
you can do
```suggestion
      userEvent.type(screen.getByRole('textbox'), 'url:"fu"');
```

Reply:
Good idea https://github.com/getsentry/sentry/pull/40289/commits/9724d55c9338b4c5f0a58b9e47854f5b61d1d183